back-end/db/models/relationships.js

```diff
@@ -15,6 +15,11 @@ Quest.belongsTo(User, {
   as: "adventurer",
 });
 
+Quest.belongsTo(Class, {
+  foreignKey: "class_id",
+  as: "class",
```

Comment:
pretty sure the `as` part is unnecessary.
```suggestion
```

Reply:
do we still want to refactor all that to match the default sequelize fk format?

Reply:
It would make things easier and terser if we were to respect sequelize's nomenclature . But I'd do that in another PR, after this one is merged.